tests/acceptance/addon-smoke-test-slow.js

```diff
@@ -99,6 +100,68 @@ if (!experiments.MODULE_UNIFICATION) {
       expect(result.code).to.eql(0);
     }));
 
+    it('works when ember-cli-build is inside tests/dummy', co.wrap(function *() {
+      yield copyFixtureFiles('addon/ember-cli-build-at-tests');
+
+      let packageJsonPath = path.join(addonRoot, 'package.json');
+      let packageJson = fs.readJsonSync(packageJsonPath);
+
+      packageJson.dependencies = packageJson.dependencies || {};
+      // add HTMLBars for templates (generators do this automatically when components/templates are added)
+      packageJson.dependencies['ember-cli-htmlbars'] = 'latest';
+
+      fs.writeJsonSync(packageJsonPath, packageJson);
+
+      let result = yield runCommand('node_modules/ember-cli/bin/ember', 'build');
+
+      expect(result.code).to.eql(0);
+
+      let cssPath = path.join(addonRoot, 'addon', 'styles', 'app.css');
+      let cssContents = fs.readFileSync(cssPath, { encoding: null });
+      let md5 = crypto.createHash('md5');
+      md5.update(cssContents);
+      let hex = md5.digest('hex');
+
+      let builtCss = path.join(addonRoot, 'dist', 'assets', `vendor-${hex}.css`);
+      let builtCssExists = fs.existsSync(builtCss);
+
+      expect(builtCssExists).to.eql(true);
+    }));
+
+    it('works when ember-cli-build is at addon root', co.wrap(function *() {
+      yield copyFixtureFiles('addon/ember-cli-build-at-root');
+
+      // remove ember-cli-build.js created at addon root
+      let dummyPath = path.join(addonRoot, 'tests', 'dummy', 'ember-cli-build.js');
+      if (fs.existsSync(dummyPath)) {
+        fs.unlinkSync(dummyPath);
```

Comment:
Why would this file be here? Doesn't this mean the fixtures are wrong?

Reply:
This is because of us creating test targets in the [before hook](https://github.com/ember-cli/ember-cli/blob/master/tests/acceptance/addon-smoke-test-slow.js#L32). This results in an `ember-cli-build.js` being present in `tests/dummy` because of my changes and since we run find-up from that directory for an addon, that file's contents are taken for this test which isn't desirable.